openedx/core/djangoapps/catalog/migrations/0003_catalogintegration_page_size.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('catalog', '0002_catalogintegration_username'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='catalogintegration',
+            name='page_size',
+            field=models.PositiveIntegerField(default=20, help_text='Maximum number of records in paginated response of a single request to catalog service.', verbose_name='Page Size'),
```

Comment:
You need to regenerate migrations - so we don't have both 0003 and 0004 for the same additional field.  You can do so by deleting both files and then doing `makemigrations` again.